src/clj/rems/service/test_data.clj

```diff
@@ -307,6 +313,7 @@
     {:default default
      :ega ega
      :decider decider
+     :decider2 decider2
```

Comment:
this could be a separate ticket as "developer" user is widely in use in test data, but agree, it is not a descriptive user and easily gets confusing double roles as handler and decider